dev-libs/hyperscan/hyperscan-5.1.1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Use repoman to check your ebuilds / to commit...

Reply:
I used repoman full -d and it said it would be out of business ;-)